docs/releasenotes/archive/service/year2020.md

```diff
@@ -5,4 +5,662 @@ sidebar_label: 2020 Archive
+
+New - We've released a new security policy you can use to set a maximum timeout for Sumo Logic UI web sessions. Sumo Logic users can set their web session timeout on the **Preferences** page, up to a maximum of 7 days. If you are a Sumo Logic admin with the **Manage Organizational Settings** role capability, you can now specify the maximum web session timeout period that users in your org can select. For more information, see [Set a Maximum Web Session Timeout](https://help.sumologic.com/Manage/Security/Set_a_Maximum_Web_Session_Timeout "Set a Maximum Web Session Timeout").
+![set-session-timeout.png](https://help.sumologic.com/@api/deki/files/10277/set-session-timeout.png?revision=1&size=bestfit&width=600&height=632)
+
+---
+## October 28, 2020 (Search)
+
+Update - Just wanted to let you know that we've made an underlying change to our [XML Parsing](https://help.sumologic.com/05Search/Search-Query-Language/01-Parse-Operators/07-Parse-XML-Formatted-Logs "Parse XML Formatted Logs") library for parsing XML-formatted logs. The following minor behavioral chances will change how results are returned from parsing XML log:
+
+1\. Empty nodes will be returned as self closing tags
+
+For example - Consider the following xml log line (see last empty tag)\
+`<users><user id="emptytag" role="manager"><first_name>Sally</first_name><last_name>Jones</last_name><email>sally@emailplace.com</email></user><user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user><user></user></users>`
+
+Query:\
+`_sourceCategory=stag/xmltest ("678" or "emptytag")\
+| limit 5\
+| parse xml "/users/user[3]" as first_name nodrop`
+
+**Old lib **would return <user></user>\
+**New lib** would return <user />\
+2\. Extra spaces in XML tags will be trimmed.
+
+For example - Consider the following xml log line (see the extra space in yellow)\
+`<users><user id="678" role="manager"><first_name>Sally</first_name><last_name>Jones</last_name><email>sally@emailplace.com</email></user><user          id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user></users>`
+
+Query:\
+`<code>_sourceCategory=stag/xmltest ("678" or "emptytag")\
+| limit 5\
+| parse xml "/users/user[2]" as first_name nodrop`
+
+**Old lib** would return: `<user          id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user>`\
+**New lib** would return: `<user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user><code>`
```

Comment:
Please fix formatting. I've done it for you here so you can see how to format code blocks and inline code:

```suggestion
1. Empty nodes will be returned as self closing tags.

For example - Consider the following xml log line (see last empty tag)\
```
<users><user id="emptytag" role="manager"><first_name>Sally</first_name><last_name>Jones</last_name><email>sally@emailplace.com</email></user><user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user><user></user></users>
```

Query:
```
_sourceCategory=stag/xmltest ("678" or "emptytag")
| limit 5
| parse xml "/users/user[3]" as first_name nodrop
```

**Old lib **would return `<user></user>`
**New lib** would return `<user />`
2. Extra spaces in XML tags will be trimmed.

For example - Consider the following xml log line (see the extra space in yellow):
```
<users><user id="678" role="manager"><first_name>Sally</first_name><last_name>Jones</last_name><email>sally@emailplace.com</email></user><user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user></users>
```

Query:
```
_sourceCategory=stag/xmltest ("678" or "emptytag")
| limit 5
| parse xml "/users/user[2]" as first_name nodrop
```

**Old lib** would return: 
```<user  id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user>
```
**New lib** would return: 
```<user id="456" role="contributor"><first_name>Bob</first_name><last_name>Smith</last_name><email>bob@emailplace.com</email></user>
```
```